ptf/tests/linerate/common.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Intel Corporation
+
+from ipaddress import IPv4Address
+
+# MAC addresses
+TREX_SRC_MAC = "b4:96:91:b4:4b:41" # Source MAC address for DL traffic
```

Comment:
It would be good to change these values with below parameters. This will reflect the configuration in upf/ptf/config/trex-cfg-for-ptf.yaml. So that it will be easy for the user to refer and set values accordingly.

TREX_SRC_MAC = "b4:96:91:b4:4b:09" # Source MAC address for DL traffic
UPF_CORE_MAC = "b4:96:91:b2:06:41" # MAC address of N6 for the UPF/BESS
UPF_ACCESS_MAC = "b4:96:91:b2:06:40" # MAC address of N3 for the UPF/BESS

Reply:
Done. Just updated the PR